subprojects/jacoco/src/main/java/org/gradle/testing/jacoco/tasks/JacocoReportBase.java

```diff
@@ -18,6 +18,7 @@
 
 import com.google.common.base.Predicate;
 import com.google.common.collect.Iterables;
+import jdk.nashorn.internal.ir.annotations.Ignore;
 import org.gradle.api.Action;
```

Comment:
```suggestion
import org.gradle.api.Action;
```

This import somehow slipped in.

Reply:
Thanks.